scijava/scijava-ops/src/main/java/org/scijava/param/ParameterStructs.java

```diff
@@ -131,10 +131,21 @@ public static Struct structOf(final Class<?> c, final Method m)
 		// NB: Reject abstract classes.
 		checkModifiers(type.getName() + ": ", problems, type.getModifiers(), true, Modifier.ABSTRACT);
 
-		// Parse class level (i.e., generic) @Parameter annotations.
+		// obatin a parameterData (preferably one that scrapes the javadoc)
```

Comment:
obtain

Reply:
I guess I fixed when I created the `MemberParser` [implementations](https://github.com/scijava/incubator/blob/cfb18013919446ef230e8ef00081b4c12e4e356d/scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/struct/FieldParameterMemberParser.java#L27)